docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10-alpine
+    volumes:
+      - ./var/data/pgsql/:/var/lib/postgresql
+    env_file: '.docker/paste/postgres_env'
+  paste:
+    build: ./.docker/paste
+    volumes:
+      - .:/app
+    depends_on:
+      - db
+  nginx:
+    build: ./.docker/nginx
+    ports:
+      - "80:80"
+    volumes:
+      - ./var/log/nginx:/var/log/nginx
+      - .:/app
+    depends_on:
```

Comment:
not a hard depends i think, you are able to restart the paste image without something breaking, but it will result in downtime. that said the time loss of restarting the nginx image is insignificant.

Reply:
I don't really know much about Docker workflows, so is it common practice to run it with `docker exec`, or are we just doing this because this is for development? Would a release build be the same way?